.pipelines/cni/singletenancy/windows-cni-load-test-template.yaml

```diff
@@ -80,8 +80,35 @@ stages:
               kubectl get pods -A
             name: "WaitForCNI"
             displayName: "Wait For CNI"
-  - stage: pod_deployment_windows
+  - stage: datapath_tests
     dependsOn: update_cni
+    displayName: "Datapath Test - Windows"
+    jobs:
+      - template: ../k8s-e2e/k8s-e2e-job-template.yaml
+        parameters:
+          clusterName: ${{ parameters.clusterName }}
+          os: ${{ parameters.os }}
+          datapath: true
+      - job: windows_datapath
+        steps:
+        - task: AzureCLI@1
+          inputs:
+            azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+            scriptLocation: "inlineScript"
+            scriptType: "bash"
+            addSpnToEnvironment: true
+            inlineScript: |
+              set -ex
+              make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+
+              # Needs to be changed after #2032 is merged
+              # test/integration/datapath/datapath_windows_test.go
+              # does not test IPv6 at this time, waiting for #2032
+              go test -count=1 test/integration/datapath/datapath_win_test.go -timeout 3m -tags connection -run ^TestDatapathWin$ -tags=connection

```

Comment:
Should we add a make command for this ? Because we will be adding this to the PR pipeline and any future change would be easy to make.